### DISCUSSION TITLE
Headless components have no styling

### DISCUSSION THREAD

**Discussion by Neamix** [tailwindlabs/headlessui#2649] [General]
Hello There 

I installed Tailwind on my Vue 3 app and it is working right but at adding Headless UI component it appears to me like there is no styling at all like that (this is a drop menu)
![image](https://github.com/tailwindlabs/headlessui/assets/85620725/22ff97cd-562b-4000-9dba-b3ac0a570efa)

**Comment by andredewaard** [comment]
This is exactly what this library tries to do and why its called 'headless'. Its only about the logic, styling is completely up to you. You can also use a library like [this](https://tailwindui.com/) that uses headlessUI to style these.
